helm/provisioner/templates/provisioner-cluster-role-binding.yaml

```diff
@@ -82,4 +82,41 @@ roleRef:
   name: local-storage-provisioner-jobs-role
   apiGroup: rbac.authorization.k8s.io
 {{- end }}
+{{- if .Values.common.podSecurityPolicy }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: local-storage-provisioner-psp-role
+  namespace: {{ .Values.common.namespace }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: {{ replace "+" "_" .Chart.Version | printf "%s-%s" .Chart.Name }}
+rules:
+- apiGroups:
+    - policy
+  resources:
+    - podsecuritypolicies
+  verbs:
+    - use
```

Comment:
should we use `resourceNames` to authorize `local-storage-provisioner-pod-security-policy` policy only?